app/components/work_history_review_component.rb

```diff
@@ -100,4 +100,17 @@ def formatted_end_date(work)
 
     work.end_date.to_s(:month_and_year)
   end
+
+  def generate_change_action(work:, attribute:)
+    if any_jobs_with_same_role_and_organisation?(work)
```

Comment:
If we didn't have this `if` branching logic, we could have used a `I18n.t('something.change_action', work: work, attribute: 'job title')` call instead of having to declare (and reuse) a `generate_change_action` method... given this, the current implementation is probably the best. 👍